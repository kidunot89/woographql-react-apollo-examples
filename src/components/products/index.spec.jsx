import { render } from '@testing-library/react';

import ProductsList from '.';

describe('ProductsList component', () => {
  const { baseElement } = render(<ProductsList />);

  it('renders successfully', () => {
    expect(baseElement).toMatchSnapshot();
  });
});
