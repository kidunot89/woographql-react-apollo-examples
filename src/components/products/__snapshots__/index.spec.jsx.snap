// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductsList component renders successfully 1`] = `
<body>
  <div />
</body>
`;
